api/new_relic.go

```diff
@@ -23,9 +23,10 @@ func newRelicApp() *newrelic.Application {
 	return app
 }
 
-func newRelicHandle(app *newrelic.Application, pth string, f func(http.ResponseWriter, *http.Request)) (string, func(http.ResponseWriter, *http.Request)) {
+func newRelicHandle(app *newrelic.Application, pth string, f http.HandlerFunc) (string, http.HandlerFunc) {
 	if app == nil {
 		return pth, f
 	}
+
```

Comment:
Just out of curiosity: is there a reason to add that blanl line?

Reply:
I imagine I added it by accident :|

Reply:
NP

I just asked to make sure I wasn't missing anything : )